### ISSUE TITLE
Spellcheck not saving work

### ISSUE THREAD

**Issue by ThisorThatguy** [worldanvil/worldanvil-bug-tracker#708]
**World Anvil Username**: StrubleBooks

**World / Campaign / RPG Character / Manuscript Title**: OLDH

**Feature**: Spellcheck

**Related URLS** (To public & visible pages add screenshots otherwise):

**Describe the Issue**
A clear and concise description of what the issue is.

I am sure you have been made aware but it seems when editing in advance mode using the spellcheck it does not always save what you are typing. This seems to happen much more frequently when you move from one section to the next. 

I was updating my character article under the personal sections and I filled out several sections when i viewed only a few random parts got saved. I lost a good amount of work :( if this is a known reported is there an pending fix?

**To Reproduce**
1. Use advance editor and spellcheck option
2. Change from one section to another
3. View work and only parts have saved

**Expected behavior**

The auto save should be saving everything in the editor as your write

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!

![84A5D347-3C17-4913-9989-28A5C4D267A2](https://user-images.githubusercontent.com/71042456/142887165-e8f5b212-7357-4bbe-8247-0511558bb024.png)

In the screenshot that sections was fully filled out (with like 2 sentences but much more than 2 word haha) this is similar in several areas. So are very odd where one sentence will be missing its end up it picked up the second sentence okay. (Not sure if I went back to it and updated the second sentence not realizing the first was lost)

I hope this helps if you have any questions feel free to contact me. 

Thank you for supporting this amazing website and community you help support!!!

**Comment by Kordishal** [comment]
When switching from Spellcheck mode back to normal mode it does take a short moment for everything changed in Spellcheck mode to apply. But I can't really make it to just disappear. My guess is you still somehow managed to make it save before the changes you made in spell check were applied and then somehow overwrote that? 

But I really can't make that happen. In general I would advice against writing in Spellcheck mode and instead just use Spell check mode to, spell check. And then after switching back make sure to wait a second until all the changes are applied properly. 

And always just work in one field at a time.

**Comment by ThisorThatguy** [comment]
Okay I think what I might have done was save while in spellcheck mode without going back to regular mode.I will try spellcheck with your suggestion (in small batches 😁) and see if it keeps happening.I did have issues with spellcheck not finding errors until I clicked on the misspelled then I would get a suggestion (I could just be impatient there).This might be an ios/tablet issue but in advance mode suggestions (from the tablet keyboard) stop showing up too when typing (figured this is due to how the editor works as they show up once in spellcheck mode).Thanks for the quick response! Hope you have a wonderful day 😊
-------- Original message --------From: Jonas Waeber ***@***.***> Date: 11/22/21  3:48 PM  (GMT-05:00) To: worldanvil/worldanvil-bug-tracker ***@***.***> Cc: ThisorThatguy ***@***.***>, Author ***@***.***> Subject: Re: [worldanvil/worldanvil-bug-tracker] Spellcheck not saving work
  (Issue #708) 
When switching from Spellcheck mode back to normal mode it does take a short moment for everything changed in Spellcheck mode to apply. But I can't really make it to just disappear. My guess is you still somehow managed to make it save before the changes you made in spell check were applied and then somehow overwrote that?
But I really can't make that happen. In general I would advice against writing in Spellcheck mode and instead just use Spell check mode to, spell check. And then after switching back make sure to wait a second until all the changes are applied properly.
And always just work in one field at a time.
